content/oc/0001-metricas-performance.md

```diff
@@ -293,20 +293,20 @@ $$
 
 :::info[Exemplo]
 
-Considerando um [processador A](color:pink) com _clock rate_ de 2GHz, e que
+Consideremos um processador $$A$$ com _clock rate_ de 2GHz, e que
 demora [10 segundos](color:orange) de tempo de CPU a executar um certo programa.  
-Queremos desenhar um [processador B](color:purple) que demore
+Queremos desenhar um processador $$B$$ que demore
 [6 segundos](color:orange) de tempo de CPU a executar o mesmo programa,
-mas que necessita de [1.2 vezes mais ciclos de relógio para executar o programa](color:orange).
-**Qual será o _clock rate_ do [processador B](color:purple)?**
+mas que necessite de [1.2 vezes mais ciclos de relógio](color:yellow).
+**Qual será o _clock rate_ do processador $$B$$?**
 
-Começando por escrever a expressão do _clock rate_ de [B](color:purple), temos:
+Começando por escrever a expressão da _clock rate_ de $$B$$, temos:
```

Comment:
Hmm, I don't think I agree with removing the colors here, I'll revert it

Reply:
Sure, that's way better! I though you couldn't combine latex with colors.